### ISSUE TITLE
Error with xcode. 

### ISSUE THREAD

**Issue by eduardo-garcia18** [shirsh94/flutter_doc_scanner#28]
Trying to use it on iOS I have the following error: 

```
Error (Xcode):
../../../.pub-cache/hosted/pub.dev/flutter_doc_scanner-0.0.10/lib/flutter_doc_scanner_web.dart:5:8:
Error: Dart library 'dart:html' is not available on this platform.
```

This is how I include it on the pubspec file: 

```
dependencies: 
  flutter_doc_scanner: ^0.0.10
```

I have try to use `flutter clean` but does not work.

**Comment by SamuelGrafstrom** [comment]
gets same error on vscode android, this is what chatgpt says: 
The issue arises because the flutter_doc_scanner package depends on the dart:html library, which is only available for web applications. Since you are building for a non-web platform (e.g., iOS, Android, or desktop), the package's dependency on dart:html is causing a build failure.

**Comment by shirsh94** [comment]
This is resolved on the 0.0.11 version. Thank you for your patience and cooperation.
